src/modules/hostname.rs

```diff
@@ -123,7 +128,7 @@ mod tests {
             })
             .env("SSH_CONNECTION", "something")
             .collect();
-        let expected = Some(format!("{} in ", style().paint(hostname)));
+        let expected = Some(format!("{} in ", ssh_style().paint(hostname)));
```

Comment:
If we're going to do this, perhaps we should use a non-standard ssh_style in the config so that we can be sure that that codepath is actually being triggered?